src/utils/checkSpelling.js

```diff
@@ -0,0 +1,110 @@
+const OriginalKeyword = require("../models/OriginalKeyword");
+
+async function getOriginalKeywords() {
+  const correctWords = await OriginalKeyword.findOne({
+    name: "originalKeywords",
+  });
```

Comment:
```suggestion
  const correctWords = await OriginalKeyword.findOne({
    name: "originalKeywords",
  }).lean();
```
불러온 도큐먼트의 값을 변경하지 않아 lean()을 붙이면 불필요한 자료의 양을 줄일 수 있어 좋을 것 같습니다.

Reply:
그렇네요! 수정하겠습니다!